ts/lib/auth/computeclient.ts

```diff
@@ -16,8 +16,18 @@
 
 import * as request from 'request';
 
+import {RequestCallback, RequestError} from './../transporters';
 import Auth2Client from './oauth2client';
 
+export interface Tokens {
+  expires_in: number;
+  expiry_date: number;
+}
+
+export interface RefreshTokenCallback {
```

Comment:
If it is used to annotate an exported member or function then it would need to be exported.